### ISSUE TITLE
ALT badge missing from quoted posts

### ISSUE THREAD

**Issue by jcsalterego** [bluesky-social/social-app#10856]
### Steps to Reproduce

1. Go to https://bsky.app/profile/jcsalterego.bsky.social/post/3mnvqsdeovk2s - note it has alt text
2. Go to quote https://bsky.app/profile/aaronblackshear.bsky.social/post/3mnxdilhzzc2s - note the quoted post's image does not have an alt text badge

Expected: quoted post image has ALT badge
Actual: quoted post image does not have an ALT badge

Reproduced on:

iOS:
```
Build version: 1.123.0.1677; Bundle info: 292ff1e (prod); Bundle date: 26060922; Platform: ios; Platform version: 26.5
```

Web:
```
Build version: 1.123.0; Bundle info: 292ff1e (prod); Bundle date: 26060922; Platform: web; Platform version: 0.0.0
```

### Attachments

[Screenshot 2026-06-10 at 4.06.16 PM.png](https://github.com/user-attachments/assets/a57d679e-66ff-41cc-af09-b9d34788c3f8)
[Screenshot 2026-06-10 at 4.08.45 PM.png](https://github.com/user-attachments/assets/2b7ee8e7-57f7-4c8d-821f-40984e46993b)

### What platform(s) does this occur on?

iOS

### Device Info

iOS and web

### What version of the app are you using?

Build version: 1.123.0.1677; Bundle info: 292ff1e (prod); Bundle date: 26060922; Platform: ios; Platform version: 26.5

**Comment by jackgopack4** [comment]
Jerry yes!

**Comment by NotThatGothic** [comment]
Can confirm this issue is also present on Android.

Build version: 1.124.0.1207; Bundle info: 6efebbd (prod); Bundle date: 26061219; Platform: android; Platform version: 33; Device ID: 8cb1fe74-10c8-4d29-b295-89b837c15333

**Comment by smenor** [comment]
Bad regression in need of a fix
